schema/dmt/schema.go

```diff
@@ -261,7 +272,7 @@ func main() {
 	ts.Accumulate(schema.SpawnString("FieldName"))
 	ts.Accumulate(schema.SpawnStruct("StructField",
 		[]schema.StructField{
-			schema.SpawnStructField("type", "TypeNameOrInlineDefn", false, false),
+			schema.SpawnStructField("type", "TypeTerm", false, false),
```

Comment:
I think that I actually wanted to carry the rename `s/TypeTerm/TypeNameOrInlineDefn/g` back _up_ to the schema-schema.  But I haven't done it yet, so if you'd rather go for consistency for today, fair 'nuff.

Reply:
Yep. Renames should go there first :) Otherwise it's going to be easy to get out of sync and forget. The source of truth is there, not here.